src/graphics/image.rs

```diff
@@ -293,3 +336,24 @@ impl Drop for Image {
         unsafe { ffi::sfImage_destroy(self.image) }
     }
 }
+
+#[derive(Debug, Copy, Clone)]
+pub enum SetPixelError {
+    XTooLarge(u32, u32),
```

Comment:
Should probably be named fields for better clarity.` {x: u32, width: u32}`

Reply:
Fixed